co2calculator/distances.py

```diff
@@ -308,3 +355,169 @@ def get_route(coords, profile: str = None) -> Kilometer:
     )  # divide my 1000, as we're working with distances in km
 
     return dist
+
+
+def _apply_detour(distance: Kilometer, transportation_mode: str) -> Kilometer:
+    """
+    Function to apply specific detour parameters to a distance as the crow flies
+    :param distance: Distance as the crow flies between location of departure and destination of a trip
+    :param transportation_mode: Mode of transport used in the trip
+    :type distance: float
+    :type transportation_mode: str
+    :return: Distance accounted for detour
+    :rtype: float
+    """
+    try:
+        detour_coefficient = detour_df[
+            detour_df["transportation_mode"] == transportation_mode
+        ]["coefficient"].values[0]
+        detour_constant = detour_df[
+            detour_df["transportation_mode"] == transportation_mode
+        ]["constant [km]"].values[0]
+    except KeyError:
+        detour_coefficient = 1.0
+        detour_constant = 0.0
+        warnings.warn(
+            f"""
+        No detour coefficient or constant available for this transportation mode.
+        Detour parameters are available for the following transportation modes:
```

Comment:
Here, we might print out detour_df["transportation_mode"] - or we just leave this out in the warning.